tests/unit/SwooleAdapterTest.php

```
@@ -0,0 +1,98 @@
+        $registeredCallback = null;
+        $mockServer->expects($this->once())
+            ->method('on')
+            ->with('message', $this->callback(function ($callback) use (&$registeredCallback) {
+                $registeredCallback = $callback;
+                return true;
+            }));
+
+        $serverProperty->setValue($adapter, $mockServer);
+
+        $callbackInvoked = false;
+        $adapter->onMessage(function (int $connection, string $message) use (&$callbackInvoked) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$connection'. (undefined)

(UnusedFormalParameter)